content/yaml/running-a-custom-script.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Running a custom script
+description: Custom scripts with YAML.
+weight: 6
+---
+
+You can run scripts in languages other than shell (`sh`) by defining the languge with a shebang line or by launching a script file present in your repository.
```

Comment:
Should rephrase this, i understand that we can run all other languages - but not shell. thats not actually true.

Reply:
fixed in 420381a